src/components/cc-ssh-key-list/cc-ssh-key-list.js

```diff
@@ -306,7 +308,13 @@ export class CcSshKeyList extends LitElement {
 
           return html`
             <div class="key ${classMap(classes)}">
-              <cc-img class="key__icon" src=${!skeleton ? keySvg : ''} ?skeleton=${skeleton}></cc-img>
+              <div class="key__icon">
+              ${
+                skeleton
+                ? html`<div class="skeleton"></div>`
+                : html`<cc-icon .icon="${IconKey}" size="lg"></cc-icon>`
+              }
+              </div>
```

Comment:
At first, I was like @pdesoyres-cc.

Then I was "in this situation, why don't we always display the icon since it's not an icon where we don't know what to display?

Then I was "we'll still need skeleton in some situations".

My conclusion is:

* maybe we should always use the icon (no skeleton) in this very example
* maybe we should also add a skeleton mode on cc-icon

Reply:
Let's create a skeleton state for `cc-icon` and see how it goes!